.github/workflows/publish_docker.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Docker image
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  docker:
+    name: Publish Docker
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Docker login
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASS }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: volkszaehler/mbmd
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
```

Comment:
```suggestion
        uses: docker/build-push-action@v6
```

Reply:
Do we need an additional

```
      - name: Setup Buildx
        uses: docker/setup-buildx-action@v3
```

?

Reply:
See lines 25 to 26. Buildx is already there.